simpletransformers/ner/ner_model.py

```diff
@@ -726,7 +726,6 @@ def evaluate(self, eval_dataset, output_dir, verbose=True, silent=False, wandb_l
 
         eval_loss = eval_loss / nb_eval_steps
         token_logits = preds
```

Comment:
Wouldn't making this change mean that we'll lose token logits here?

Reply:
Sorry, missed that. That's certainly an issue. I think that having a fast evaluate function that only does evaluation would be super useful. What about adding a new argument to `evaluate` (e.g. `return_logits=False`) to determine if the logits should be returned, and if not `results, None , preds_list` will be returned instead?

Reply:
I can see the value in that. Perhaps both the `evaluate` and the `predict` functions should have the `return_logits` flag?

Another issue here is that `np.append()` is being called in the loop. Creating the array beforehand and inserting the values in the loop is more efficient. I've already done this for the `ClassificationModel`, and plan on doing it for `NER` as well.

Reply:
- Added `return_logits` flag to `evaluate` and `eval_model`.
- Created the predictions array before the loop instead of relying on `np.append()`.